crates/types/src/color.rs

```diff
@@ -147,25 +140,27 @@ impl Rgb {
     pub const TURQUOISE: Rgb = Rgb::new(0, 255, 255);
     pub const WHITE: Rgb = Rgb::new(255, 255, 255);
 
-    pub const fn new(r: u8, g: u8, b: u8) -> Self {
-        Self { r, g, b }
+    pub const fn new(red: u8, green: u8, blue: u8) -> Self {
+        Self { red, green, blue }
     }
 
-    pub fn get_chromaticity(&self, channel: RgbChannel) -> f32 {
-        let sum = self.r as f32 + self.g as f32 + self.b as f32;
-        if sum == 0.0 {
-            return 0.0;
+    pub fn convert_to_rgchromaticity(&self) -> RgChromaticity {
+        let sum = self.red as f32 + self.green as f32 + self.blue as f32;
```

Comment:
```suggestion
let sum = (self.red + self.green + self.blue) as f32;
```

Reply:
done

Reply:
if I do this it would add u8 to each other, which would be bigger than 255

Reply:
wow, stupid of me, of course you are right